NeZha-D1s/README.md

```diff
@@ -16,22 +16,26 @@ cpu_core: XuanTie C906
 - RT-Thread:
   - Download Link: https://github.com/RT-Thread/rt-thread
   - Reference Installation Document: https://github.com/RT-Thread/rt-thread/blob/master/bsp/allwinner/d1s/README-M7.md
-- Ubuntu 24.04:
-  - Download Link: https://cdimage.ubuntu.com/releases/noble/release/ubuntu-24.04-preinstalled-server-riscv64+nezha.img.xz?
-  - Reference Installation Document: https://wiki.ubuntu.com/RISC-V/Nezha%20D1?
+- Ubuntu 24.10:
+  - Download Link: https://cdimage.ubuntu.com/releases/24.10/release/
+  - Reference Installation Document: https://wiki.ubuntu.com/RISC-V/Nezha%20D1
+- Ubuntu 24.04.1 LTS:
+  - Download Link: https://cdimage.ubuntu.com/releases/24.04.1/release/
+  - Reference Installation Document: https://wiki.ubuntu.com/RISC-V/Nezha%20D1
 
 ### Hardware Information
 
 - D1s NeZha
 
 ## Test Results
 
-| Software Category     | Package Name | Test Results (Test Report) |
-| --------------------- | ------------ | -------------------------- |
-| Tina-Linux Image Boot | N/A          | [CFT][Tina]                |
-| RT-Thread Image Boot  | N/A          | [CFT][RT-Thread]           |
-| Ubuntu Image Boot     | N/A          | [Success][Ubuntu]          |
+| Software Category     | Package Name | Test Results (Test Report)   |
+| --------------------- | ------------ | ---------------------------- |
+| Tina-Linux Image Boot | N/A          | [CFT][Tina]                  |
+| RT-Thread Image Boot  | N/A          | [CFT][RT-Thread]             |
+| Ubuntu Image Boot     | N/A          | [Success][Ubuntu][Ubuntu LTS]|
```

Comment:
You're sure to write it like this?
`[aaa][bbb]` and then `[bbb]: url` at the bottom means to reference the link in `bbb` and use it for `aaa`, while only displaying `aaa`.

Reply:
sorry, my mistakes

Reply:
sorry, my mistakes